docs/userdoc/AutoTuning.md

```diff
@@ -0,0 +1,46 @@
+# AutoTuning
+
+One of the primary features of AutoPas is the dynamic automated algorithm selection for the force calculation.
+Since AutoPas follows the philosophy of [Convention over Configuration](https://en.wikipedia.org/wiki/Convention_over_configuration), it will automatically tune itself to find the optimum of all its internal algorithm configurations.
+However, this naive tuning is just an exhaustive search by trialling all algorithms over successive iterations.
+While this is guaranteed to find the optimum, it is very costly and inefficient.
+Therefore, AutoPas offers a bunch of [Tuning Strategies](https://github.com/AutoPas/AutoPas/blob/master/src/autopas/options/TuningStrategyOption.h) that can be applied and combined for an optimal experience.
+
+## Tuning Loop
+To better understand what the strategies do, we first take a look at the way AutoPas does its tuning.
+
+The basic idea is that the AutoTuner class determines a configuration, which the LogicHandler then assembles and applies to the ParticleContainer.
+In order to determine the optimal configuration, the AutoTuner has a queue of candidates that it tries out by using it for one interaction computation in order to measure their performance.
+Those iterations while candidate configurations are evaluated are referred to as tuning phase.
+AutoPas works on the assumption that particle simulations evolve very slow and thus the simulation state of subsequent iterations is sufficiently similar to compare the configurations' performances.
+When the queue is fully processed, the configuration with the optimal measurement is determined to be the current algorithm optimum and is used until the start of the next tuning phase.
+
+### Tuning Interval
+The interval between the start of two tuning phases can be configured by the user.
+Iterations for sampling the configurations do count against this interval so if an tuning interval of 1000 is set tuning will start at iterations 0, 1000, 2000.
+Should the tuning take longer than one interval any starts that fall into an active tuning phase are skipped.
+
+### Tuning Strategies
+By default, the AutoTuner configuration queue is filled with all applicable configurations and processed sequentially.
+Tuning strategies can filter, modify, and reorder this queue.
+After each sampling of a configuration, they receive the measurement, called evidence, and some live information about the current state of the domain.
+Multiple strategies can be activated at the same time.
+They are all applied in every decision process in the order they are specified in the vector of `ConfigurationOption`s passed to AutoPas.
```

Comment:
What is `ConfigurationOption` passed to AutoPas? Do you mean which strategies are enabled when starting the simulation?

Reply:
My bad it was meant to be like this:
```suggestion
They are all applied in every decision process in the order they are specified in the vector of `TuningStrategyOption`s passed to AutoPas.
```